src/engine/DrawManager.java

```diff
@@ -623,39 +628,79 @@ public void drawItemSelectingTitle(final Screen screen, final GameState gameStat
 
 	}
 
-	public void drawSelectedItem(final Screen screen, final List<Integer> itemList, final int selectedItem) {
+	public void drawSelectedItem(final Screen screen, final List<Item> itemList, final int selectedItem) {
+		if (itemList.size() < 3) {
+			String alarmString = "Only  " + itemList.size() + "  items remaining!!";
```

Comment:
```suggestion
			String alarmString = "Only  " + itemList.size() + "  items left you can upgrade!!";
```
그냥 '남았다'고 표현하기 보다는 업그레이드 가능한 아이템의 수가 이만큼 남았다고 알려 주는 게 좋을 것 같습니다.